device/api/umd/device/cluster.h

```diff
@@ -648,13 +634,12 @@ class Cluster : public tt_device {
     virtual bool using_harvested_soc_descriptors();
     virtual std::unordered_map<chip_id_t, uint32_t> get_harvesting_masks_for_soc_descriptors();
     virtual void translate_to_noc_table_coords(chip_id_t device_id, std::size_t& r, std::size_t& c);
-    virtual int get_number_of_chips_in_cluster();
```

Comment:
Is this function not being used or we just don't want to support this interface since client can always call 
```
int num = get_target_device_ids().size();
```

Reply:
Both.
Also not `get_target_device_ids().size()` but rather `detect_available_device_ids().size()`, see previous comment :) It adds to confusion I guess.